spec/requests/posts_controller_spec.rb

```diff
@@ -406,6 +406,18 @@
         expect(response.status).to eq(200)
       end
 
+      it "triggers DiscourseEvent with :posts_destroyed and correct params" do
+        sign_in(poster)
+        # Use spy to monitor DiscourseEvent.trigger
+        allow(DiscourseEvent).to receive(:trigger).and_call_original
```

Comment:
We have a nice clean way of tracking events like this -> https://github.com/discourse/discourse/blob/f291452c9a53f4d3318351585799282b77902366/spec/lib/post_action_creator_spec.rb#L129